API/Supported_Languages.md

```diff
@@ -0,0 +1,31 @@
+# Supported languages
+
+## JavaScript/Node
+Tessel 2 has full support for JavaScript and Node.JS. The relevant repos can be found here:
+
+* [Firmware support for JavaScript/Node](https://github.com/tessel/t2-firmware)
+* [CLI support for JavaScript/Node](https://github.com/tessel/t2-cli/blob/master/lib/tessel/deployment/javascript.js)
+
+### Binary modules
+There is some support for modules with binary dependencies on Tessel 2.
+
+The best way to find out whether the module you want is supported is to try it. If it's supported, it will work! If it is not supported, you will get an error message like this ([Error message source in CLI](https://github.com/tessel/t2-cli/blob/master/lib/tessel/deployment/javascript.js)):
```

Comment:
"The best way to find out whether the module you want is available is to try deploying it."